src/apps/ociocheck/main.cpp

```diff
@@ -209,6 +209,40 @@ int main(int argc, const char **argv)
             for(int i=0; i<config->getNumLooks(); ++i)
             {
                 std::cout << config->getLookNameByIndex(i) << std::endl;
+
+                OCIO::ConstLookRcPtr look = config->getLook(config->getLookNameByIndex(i)); 
+
+                OCIO::ConstTransformRcPtr transform = look->getTransform();         
+                OCIO::ConstTransformRcPtr invTransform = look->getInverseTransform();         
```

Comment:
[Coding Style] You should keep close related code. This line could be moved at line 231 ( while preserving the empty line from the ending curly bracket i.e. `}` )